src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IOrchestrationFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.orchestration('DurableFunctionsOrchestratorJS', function* (context: IOrchestrationFunctionContext) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity('Hello', 'Tokyo'));
+    outputs.push(yield context.df.callActivity('Hello', 'Seattle'));
+    outputs.push(yield context.df.callActivity('Hello', 'Cairo'));
+
+    return outputs;
+});
+
+df.activity('Hello', (context: InvocationContext, name: string) => {
```

Comment:
Rename `name` to `input`. This would match the name of the arg in `callActivity` and will work for more realistic scenarios

Reply:
Done